roles/smartgateway/tasks/main.yml

```diff
@@ -4,37 +4,30 @@
   set_fact:
     api_groups: "{{ lookup('k8s', cluster_info='api_groups') }}"
 
-- name: "Set Smart Gateway container images"
+- name: Set Smart Gateway core image
+  set_fact:
+    core_container_image_path: "{{ lookup('env', 'CORE_SMARTGATEWAY_IMAGE') | default('quay.io/infrawatch/sg-core:latest', true) }}"
```

Comment:
Can we change this these env vars to be prefixed with `RELATED_IMAGE_`? This is required for downstream imports and it'd be good to just get this adjusted now.

You'll need to adjust this in both the CSV and the Ansible role like you're doing here.

e.g. `RELATED_IMAGE_CORE_SMARTGATEWAY_IMAGE`

Reply:
done